library/Zend/Mail/Transport/SendgridSmtp.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class WN_Mail_Transport_SendgridSmtp extends WN_Mail_Transport_Smtp
+{
+    /**
+     * ClickTracking status
+     *
+     * @var bool
+     */
+    protected $clickTracking = false;
```

Comment:
shouldn't this be private?

Reply:
my thought was just if someone wanted to extend it, they could do that via the constructor though